scripts/generate-stylesheet.ts

```diff
@@ -0,0 +1,48 @@
+import { ColorRegistry } from '../packages/main/src/plugin/color-registry';
```

Comment:
missing copyright headers and the header to execute the file directly
```
#!/usr/bin/env tsx
```

Reply:
Applied